eng/common/templates/job/performance.yml

```diff
@@ -7,7 +7,7 @@ parameters:
   container: ''                   # required -- name of the container
   osGroup: ''                     # required -- operating system for the job
   extraSetupParameters: ''        # optional -- extra arguments to pass to the setup script
-  frameworks: ['netcoreapp3.0']   # optional -- list of frameworks to run against
+  frameworks: ['netcoreapp3.1']   # optional -- list of frameworks to run against
```

Comment:
Everything under `eng/common` is owned by Arcade.  Soon we'll get an auto-PR that reverts this line.  No idea on what'll happen, though.

Reply:
Oops, didn't notice this. Any concern that it'll cause misbehavior?